SimPEG/directives/directives.py

```diff
@@ -1669,6 +1670,7 @@ def getWr(self):
 
             wr = wr ** 0.5
             wr /= wr.max()
+            wr[wr < self.truncation_factor] = self.truncation_factor
```

Comment:
Isn't the same thing as `threshold`, just moved outside of the sum? I would suggest just using the `threshold` parameter for less refactoring required.

Reply:
they are really doing the same thing, just a smoother corner if you add. 

I am cool with just moving out the threshold out of the sum, but not a fan of adding another term...

Reply:
I see your point, BUT you don't know if the largest value in diag(JtJ) is 1000 or 0.0001. For example, if I set the threshold to be 1e-6 and my largest diag(JtJ) value is 1e-8, my sensitivity weights are essentially just a constant model of 1e-6.

Whether or not we use threshold or truncation factor, it needs to be done AFTER normalizing and not before!!!

Reply:
Agreed.